test/e2e/specs/wp-likes-spec.js

```diff
@@ -0,0 +1,124 @@
+			await this.loginFlow.loginAndStartNewPost( null, true );
+
+			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+			await gEditorComponent.enterTitle( blogPostTitle );
+			await gEditorComponent.enterText( blogPostQuote );
+			postUrl = await gEditorComponent.publish( { visit: true } );
+		} );
+
+		step( 'Like post', async function () {
+			await PostAreaComponent.Expect( driver );
+
+			const iFrame = By.css( 'iframe.post-likes-widget' );
+			const likeButton = By.css( '.like.sd-button' );
+			const postLikedText = By.xpath( `//span[@class='wpl-count-text'][.='You like this.']` );
+
+			await driver.switchTo().defaultContent();
+			await driverHelper.waitTillPresentAndDisplayed( driver, iFrame );
+			await driverHelper.waitTillAbleToSwitchToFrame( driver, iFrame );
+			await driverHelper.scrollIntoView( driver, likeButton );
+			await driverHelper.clickWhenClickable( driver, likeButton );
+			await driverHelper.waitTillPresentAndDisplayed( driver, postLikedText );
+			await driver.switchTo().defaultContent();
+		} );
+
+		step( 'Post and like comment', async function () {
+			const commentArea = await CommentsAreaComponent.Expect( driver );
+
+			const comment = dataHelper.randomPhrase();
+
+			await commentArea._postComment( {
+				comment: comment,
+				name: 'e2eTestName',
+				email: 'e2eTestName@test.com',
+			} );
+
+			const commentLikeLink = By.xpath(
+				`//div[@class='comment-content']/p[.='${ comment }']/../p/a[@class='comment-like-link']`
+			);
+			const commentLikedText = By.xpath(
+				`//div[@class='comment-content']/p[.='${ comment }']/../p/span[starts-with(text(),'Liked by')]`
+			);
+
+			await driver.switchTo().defaultContent();
+			await driverHelper.scrollIntoView( driver, commentLikeLink, 'end' );
+			await driverHelper.clickWhenClickable( driver, commentLikeLink );
+			await driverHelper.waitTillPresentAndDisplayed( driver, commentLikedText );
+		} );
+
+		describe( 'Like from logged out', function () {
+			step( 'View profile to log out', async function () {
+				const navbarComponent = await NavBarComponent.Expect( driver );
+				await navbarComponent.clickProfileLink();
+			} );
+
+			step( 'Logout from profile page', async function () {
+				const profilePage = await ProfilePage.Expect( driver );
+				await profilePage.clickSignOut();
+			} );
+
+			step( 'See wordpress.com home when after logging out', async function () {
+				return await LoggedOutMasterbarComponent.Expect( driver );
+			} );
+
+			step( 'Like post as logged out user', async function () {
+				const iFrame = By.css( 'iframe.post-likes-widget' );
+				const postLikesArea = new AsyncBaseContainer( driver, iFrame, postUrl );
+				postLikesArea._visitInit();
+
+				const likeButton = By.css( '.like.sd-button' );
+				const postLikedText = By.xpath( `//span[@class='wpl-count-text'][.='You like this.']` );
+
+				await driver.switchTo().defaultContent();
+				await driverHelper.waitTillPresentAndDisplayed( driver, iFrame );
```

Comment:
Same as https://github.com/Automattic/wp-calypso/pull/51618/files#r607733502

Reply:
I think they're there for slightly different implementation reasons but I'll try to remove it and see if it still behaves correctly. Still quite new to these e2e tests so thanks for catching these and pointing them out!